rdbms/src/main/java/org/spine3/server/storage/jdbc/command/query/InsertCommandQuery.java

```diff
@@ -20,13 +20,14 @@
 
 package org.spine3.server.storage.jdbc.command.query;
 
-import org.spine3.server.storage.CommandStorageRecord;
+import org.spine3.server.command.storage.CommandStorageRecord;
 
 import static org.spine3.server.storage.jdbc.command.query.CommandTable.COMMAND_COL;
```

Comment:
I do want to have a separate PR for that. Now the latest version of `JDBC-storage` is `0.7.0` which is way outdated. To make it compatible with the release `v0.8.0` core, I've created this particular patch.